CRM/CloseAccountingPeriod/Form/PriorFinancialPeriod.php

```diff
@@ -61,9 +68,9 @@ public function setDefaultValues() {
   public function buildQuickForm() {
     $priorFinancialPeriod = CRM_CloseAccountingPeriod_BAO_CloseAccountingPeriod::getPriorFinancialPeriod($this->_contactID);
     if ($priorFinancialPeriod) {
-      $this->assign('priorFinancialPeriod', $priorFinancialPeriod);
+      $this->assign('priorFinancialPeriod', $priorFinancialPeriod['M'] . '-' . $priorFinancialPeriod['Y']);
     }
-    $this->addDate('prior_financial_period', ts('Prior Financial Period'), TRUE);
+    $this->add('date', 'prior_financial_period', ts('Prior Financial Period'), CRM_Core_SelectValues::date(NULL, 'M Y', 2, 5), TRUE);
```

Comment:
@JoeMurray that wouldn't work here since we need a format that the date widget recognises. In this case, the 

`CRM_Core_SelectValues::date(NULL, 'M Y', 2, 5)`

returns

```(
    [addEmptyOption] => 1
    [emptyOptionText] => - select -
    [emptyOptionValue] => 
    [format] => M Y
    [smarty_view_format] => %B %Y
    [time] => 
    [minYear] => 2015
    [maxYear] => 2022
)```